Server/Services/googleService.js

```diff
@@ -9,7 +9,6 @@ module.exports = {
         //read in DB
         firebaseFunctions.getDataFromFireBase(uid, 'GoogleService')
         .then(data => {
-
             //required to obtain an access token
```

Comment:
I doesn't see it before, but you can improve the documentation by using:
```
/**
* @brief
* /
```
instead of
```
//
```

Reply:
done